MediaSession.cpp

```diff
@@ -173,7 +183,12 @@ MediaKeySession::MediaKeySession(const uint8_t *f_pbInitData, uint32_t f_cbInitD
     , m_piCallback(nullptr) {
   DRM_RESULT dr = DRM_SUCCESS;
   DRM_ID oSessionID;
+
+#ifdef PR_3_3
+  DRM_DWORD cchEncodedSessionID = sizeof(m_rgchSessionID);
+#else
   DRM_DWORD cchEncodedSessionID = SIZEOF(m_rgchSessionID);
```

Comment:
@wouterlucas SIZEOF, ZEROMEM and MEMCPY aren't present in our porting kit release. Is there a reason we can't update the code to just use memset, memcpy and sizeof?

Reply:
No reason that I can think of, I'm OK with doing it as above for now.